api/allocations.go

```diff
@@ -84,7 +84,7 @@ func (a *Allocations) Info(allocID string, q *QueryOptions) (*Allocation, *Query
 //
 // The call blocks until command terminates (or an error occurs), and returns the exit code.
 func (a *Allocations) Exec(ctx context.Context,
-	alloc *Allocation, task string, tty bool, command []string,
+	alloc *Allocation, task string, tty bool, command, envs []string,
```

Comment:
```suggestion
	alloc *Allocation, task string, tty bool, command []string, envs []string,
```

Reply:
Go allows skipping redundant type declaration when multiple func parameters have the same type, so the code is ok as-is.